helm/acapy-cloud/conf/local/did-registrar.yaml

```
@@ -0,0 +1,27 @@
+fullnameOverride: did-registrar
+
+replicaCount: 1
+
+podAnnotations:
+  sidecar.istio.io/proxyCPU: 10m
+
+image:
+  registry: docker.io
+  name: universalregistrar/uni-registrar-web
+  pullPolicy: Always
+  digest: sha256:d02207a156cb97dd79d3515bc90a963d12660c4ca82382f77ae6b5d32351267f
```

For multi-arch this will need to be updated to the next release or a new sha on `latest` tag as https://github.com/decentralized-identity/universal-registrar/issues/88 has been resolved

---

After some preliminary local testing (creating a `did:cheqd` and a Schema), I think we can bump this:
```suggestion
image:
  registry: docker.io
  name: universalregistrar/uni-registrar-web
  pullPolicy: Always
  tag: latest
  digest: sha256:f94053ab9b7ee8104f88bc9ecceeb0353455a9559ec1ab933f9fab7588d67f16
```